lib/container.ts

```diff
@@ -0,0 +1,30 @@
+export interface CanRender {
+  render: (element: HTMLElement) => any;
+}
+
+export interface CanAdd {
+  add: (element: CanRender) => any;
+}
+
+export const Container = function (tag?: string): CanRender & CanAdd {
+  if (!tag) {
+    tag = "div";
+  }
+
+  let self = {
```

Comment:
If a variable is not reassigned, it should be declared as `const`
(counts for other places as well)

Reply:
I wanted to change as little as needed here to not make the changes more confusing than they  already are but can change it at least for the self's